src/app/utils/chart-utils.ts

```diff
@@ -64,6 +64,16 @@ export const getMonthlyBucketsDailyAverage = (buckets: Buckets): Buckets => {
   }))
 }
 
+export const getTotalTxValues = (buckets: Buckets): Buckets => {
+  return buckets?.reduce((acc: TxVolume[], cur, index) => {
+    acc.push({
+      bucket_start: cur.bucket_start,
+      tx_volume: index > 0 ? cur.tx_volume + acc[index - 1].tx_volume : cur.tx_volume,
+    })
+    return acc
+  }, [])
+}
```

Comment:
I think this would be much more readable:
```ts
export const getTotalTxValues = (buckets: Buckets): Buckets => {
  let acc = 0
  return buckets?.map(bucket => {
    acc += bucket.tx_volume
    return {
      bucket_start: bucket.bucket_start,
      tx_volume: acc,
    }
  })
}
```

and it can be generalized:
```ts
export function cumulativeSum<T>(
  items: T[],
  accumulateField: { [F in keyof T]: T[F] extends number ? F : never }[keyof T],
): T[] {
  let acc = 0
  return items.map(item => {
    acc += item[accumulateField] as number
    return { ...item, [accumulateField]: acc }
  })
}

buckets ? cumulativeSum(buckets, 'tx_volume') : undefined
```